mlos_core/mlos_core/optimizers/optimizer.py

```diff
@@ -150,7 +134,7 @@ def suggest(
         *,
         context: Optional[pd.DataFrame] = None,
```

Comment:
```suggestion
        context: Optional[pd.Series] = None,
```

Did this one get replacd by a Series elsewhere?